compiler/src/iree/compiler/Codegen/LLVMCPU/Passes.cpp

```diff
@@ -532,6 +532,11 @@ void addMultiTilingExpertPassPipeline(OpPassManager &passManager,
         createLinalgSingleTilingExpertPass(options));
     nestedModulePM.addNestedPass<func::FuncOp>(createCanonicalizerPass());
     nestedModulePM.addNestedPass<func::FuncOp>(createCSEPass());
+    // TODO(hanchung): Merge two vectorization passes into a pass. All the ops
+    // should be vectorized altogether. Otherwise, there would be tensor.empty
+    // ops which becomes a stack allocation in bufferization.
```

Comment:
What are the two vectorization passes? Do you mean moving this to the main vectorization pass?

Reply:
One is triggered by CodegenStrategyDriver, and the other one is createVectorizePackUnPackOpsPass. The latter one actually has the main vectorization patterns, but it does not support other tricks like peeling.

I don't want to impact existing codegen and pack ops are still under experiments, so I add the pass right after it. I'm fine with taking the light overheads for now.

The other point is that I don't want to make codegen strategy more complicated since we've had discussion about retiring the CodegenStrategyDriver things.